### ISSUE TITLE
[MYSQL] Interrupted 'import-data' restarted from beginning for table with Primary Key

### ISSUE THREAD

**Issue by shubham-yb** [yugabyte/yb-voyager#311]
After I interrupted the `import-data` command in between for around 200000 rows on MYSQL, instead of resuming from the last point, it restarted from the beginning after throwing this statement.

`truncate table 'MOCK_DATA' with NO Primary Key for import of data to restart from beginning...`

Even though my table had a Primary Key. Branch used : v1.0.0-xbeta

This is the schema.

```
create table MOCK_DATA (
	id int primary key auto_increment,
	first_name VARCHAR(50),
	last_name VARCHAR(50),
	email VARCHAR(50),
	gender VARCHAR(50),
	ip_address VARCHAR(20)
);
```

This is the description on the source.

![Image](https://user-images.githubusercontent.com/65844208/189043629-b0adf766-9cb8-4942-89d6-31537b083b7d.png)

This is the description on the target.

![Image](https://user-images.githubusercontent.com/65844208/189043679-26159fd6-94ec-4f8f-90db-b67fa55d8ea7.png)

**Comment by sanyamsinghal** [comment]
Thanks @shubham-yb 

The issue is in the code where we check the primary key presence. 
The table names in MySQL are Case sensitive by default, so it was like:
MOCK_DATA -> MySQL
mock_data -> YugabyteDB
but the query was checking for MOCK_DATA in YugabyteDB, which is even not present in the target.

Fix:
1. In import data, if the table names in unquoted then convert to the small case before comprising it in system catalog tables, where if quoted just use it as it.
2. Error out in case the exact table is not present in target while checking for PK presence.

**Comment by priyanshi-yb** [comment]
resolved by #332
